torchao/float8/README.md

```diff
@@ -269,3 +269,37 @@ python test/float8/test_fsdp2/test_fsdp2.py
 # make sure to turn on torch.compile to get the best performance
 ./benchmarks/float8/bench_linear_float8.py -o ../tmp/test.txt --compile
 ```
+
+### Training benchmarks
+
+[Torchtitan](https://github.com/pytorch/torchtitan) was used to benchmark float8 training performance, for both rowwise
+and tensorwise scaling. The training benchmarks were all run using:
+
+- Single-node training on 8xH100 GPUs
+- Batch size 2
+- Sequence length 8192
+- Steps 100
+- `torch.compile`
+- FSDP2
+
+| Model         | Scaling     | Activation checkpointing | Average tokens/second     | Peak Memory (GB) |
+| ------------- | ----------- | ------------------------ | ------------------------- | ---------------- |
+| Llama3-8b     |  tensorwise | per op SAC               | 3217.4                    | 75.47            |
+| Llama3-8b     |  rowwise    | per op SAC               | 2838.1                    | 75.55            |
+
+In these benchmarks tensorwise scaling achieved ~13% higher tokens/second over rowwise scaling. However, it is important to note
+that rowwise scaling has been shown to yield improvments in training loss/accuracy due to reduced quantization error, particularly
+when training large models for many steps.
+
+**Reproducing training benchmarks**
+To reproduce these benchmarks, you can follow these steps:
+
+1. On a machine with 8 H100 GPUs, clone torchtitan and follow local installation [steps](https://github.com/pytorch/torchtitan?tab=readme-ov-file#installation),
+including [downloading a tokenizer](https://github.com/pytorch/torchtitan?tab=readme-ov-file#downloading-a-tokenizer).
+2. Install torchao following these [steps](https://github.com/pytorch/ao/tree/main?tab=readme-ov-file#installation).
+2. From the torchtitan root directory, you can run the following command to reproduce the benchmarks. Note
```

Comment:
this is great!  IMO we should mention that the syntax of how to reproduce these from torchtitan may change (from torchtitan, if they change their arguments), and include the torch, torchao, torchtitan versions used to generate these.

Reply:
I created a benchmarking script to automate this and added a short guide for it: https://github.com/pytorch/ao/pull/1802

updated this PR to use it